CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Next
 
+* [PR #296]: Changed the app store link (Issue 288)
```

Comment:
I think it would be nice to add that it's necessary to change the `MOBILE_APP_STORE_PAGE_IOS` value too